### PR TITLE
AER-418 ADMS diurnal variation IMAER, pt 2

### PR DESCRIPTION
**Pull request by BertScholten**
Added diurnal variation to ADMS objects in IMAER (EAP and XSD).
Ensuring roundtriptest for ADMS source characteristics work.

TODO: road part (that needs some changes in domain object, and since I'm not working the rest of the week it seemed like a good idea to get this PR going...)

### CODE REVIEW

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/CustomDiurnalVariationType.java [hunk 1]
```diff
@@ -33,13 +33,13 @@ public enum CustomDiurnalVariationType {
...
-    public int sum(final List<Integer> values) {
+    public double sum(final List<Double> values) {
       // Special case here: the first day (24 values) should be multiplied by 5.
       if (values.size() != getExpectedNumberOfValues()) {
         throw new IllegalArgumentException("Unexpected number of values for 3-day profile: " + values.size());
       }
-      final int sumWeekdays = values.subList(0, 24).stream().mapToInt(x -> x * 5).sum();
-      final int sumOtherDays = values.subList(24, values.size()).stream().mapToInt(x -> x).sum();
+      final double sumWeekdays = values.subList(0, 24).stream().mapToDouble(x -> x * 5).sum();
+      final double sumOtherDays = values.subList(24, values.size()).stream().mapToDouble(x -> x).sum();
```
Comment (JornC) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/CustomDiurnalVariationType.java:42]: These can still be ints no?

**Comment by JornC** [comment]
Why can't all the ints not remain ints, that way the backend effectively doesn't have to deal with doubles/rounding until sending it off to ADMS?

**Review by JornC — APPROVED** [review]
ACK

**Review by Hilbrand — APPROVED** [review]
LGTM

**Pull request merged**